src/EnergyPlus/DataAirSystems.hh

```diff
@@ -90,18 +91,18 @@ namespace DataAirSystems {
     struct AirLoopCompData // data for an individual component
     {
         // Members
-        std::string TypeOf;          // The 'keyWord' identifying  component type
-        std::string Name;            // Component name
-        int CompType_Num;            // Numeric designator for CompType (TypeOf)
-        int CompIndex;               // Component Index in whatever is using this component
-        HVACSystemData *compPointer; // pointer to HVAC system
-        int FlowCtrl;                // Component flow control (ACTIVE/PASSIVE)
-        bool ON;                     // When true, the designated component or operation scheme is available
-        bool Parent;                 // When true, the designated component is made up of sub-components
-        std::string NodeNameIn;      // Component inlet node name
-        std::string NodeNameOut;     // Component outlet node name
-        int NodeNumIn;               // Component inlet node number
-        int NodeNumOut;              // Component outlet node number
+        std::string TypeOf;                        // The 'keyWord' identifying  component type
```

Comment:
Probably not part of this PR, but `std::string TypeOf` is redundant with `SimAirServingZones::CompType CompType_Num`. Only the second of these two should be needed.

Reply:
I got it.